.github/workflows/update_issues.yml

```diff
@@ -0,0 +1,50 @@
+name: update issues
```

Comment:
It would be nice to have a more descriptive file name and workflow name.... What kind of issues is it updating? This sounds like it is updating our issues in the hub, which is confusing...